exporter/stats/stackdriver/stackdriver.go

```diff
@@ -189,26 +191,36 @@ func (e *Exporter) makeReq(vds []*stats.ViewData) *monitoringpb.CreateTimeSeries
 	}
 }
 
+// createMeasure creates a MetricDescriptor for the given view data in Stackdriver Monitoring.
+// An error will be returned if there is already a metric descriptor created with the same name
+// but it has a different aggregation, window or keys.
 func (e *Exporter) createMeasure(ctx context.Context, vd *stats.ViewData) error {
 	e.createdViewsMu.Lock()
 	defer e.createdViewsMu.Unlock()
 
 	m := vd.View.Measure()
 	agg := vd.View.Aggregation()
 	window := vd.View.Window()
+	tagKeys := vd.View.TagKeys()
 	viewName := vd.View.Name()
 
-	_, ok := e.createdViews[viewName]
-	if ok {
+	if md, ok := e.createdViews[viewName]; ok {
+		// Check agg, window and keys.
+		if err := equalAggWindowTagKeys(md, agg, window, tagKeys); err != nil {
+			return err
+		}
 		return nil
 	}
 
-	merticName := monitoring.MetricMetricDescriptorPath(e.o.ProjectID, namespacedViewName(viewName, true))
-	_, err := e.c.GetMetricDescriptor(ctx, &monitoringpb.GetMetricDescriptorRequest{
-		Name: merticName,
+	metricName := monitoring.MetricMetricDescriptorPath(e.o.ProjectID, namespacedViewName(viewName, true))
+	md, err := e.c.GetMetricDescriptor(ctx, &monitoringpb.GetMetricDescriptorRequest{
+		Name: metricName,
 	})
+	if err := equalAggWindowTagKeys(md, agg, window, tagKeys); err != nil {
```

Comment:
This call to equalAggWindowTagKeys should be moved inside the "if err == nil" block below.

Reply:
I'll complete this part of the review @jcd2, if it is okay with you @rakyll.

Reply:
This was fixed by @songy23 in PR https://github.com/census-instrumentation/opencensus-go/pull/205 so it is all done.